src/types/onyx/Task.ts

```diff
@@ -0,0 +1,27 @@
+import Report from './Report';
+
+type Task = {
+    /** Title of the Task */
+    title: string;
+
+    /** Description of the Task */
+    description?: string;
+
+    // TODO: Make sure this field exists in the API
```

Comment:
@blazejkustra This field does not exist in the API or the database from what I can tell. What should we do with this TODO?

Also, please don't add TODOs to the code. Instead, open a GH so that there is a very actionable way to track what needs to happen.

Reply:
Ahh sorry, this was quite a big PR and I forgot about this TODO, so this kind of proves your point 👍 

Could you create an issue to remove `shareDestination` from the E/App? It is used in various places, and if `shareDestination` doesn't come from API/database we should be able to safely remove some of the logic:
![image](https://github.com/Expensify/App/assets/46095609/2b3e027d-6e28-423d-86bd-11a616a09133)

Reply:
[Created the issue here](https://github.com/Expensify/App/issues/29402)